pkg/antctl/raw/set/flowaggregator/command_test.go

```diff
@@ -0,0 +1,298 @@
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "testConfigMap",
+			Namespace: "flow-aggregator",
+		},
+	}
+
+	k8sClient := fake.NewSimpleClientset(faConfigMap)
+	tcs := []struct {
+		name          string
+		configMapName string
+		podNamespace  string
+		args          []string
+		expectedErr   string
+	}{
+		{
+			name:          "valid configMap and nil args",
+			configMapName: "testConfigMap",
+			podNamespace:  "flow-aggregator",
+		},
+		{
+			name:          "invalid configMap and nil args",
+			configMapName: "",
+			podNamespace:  "flow-aggregator",
+			expectedErr:   "failed to locate flow-aggregator-config ConfigMap volume",
+		},
+		{
+			name:          "valid configMap and valid args",
+			configMapName: "testConfigMap",
+			podNamespace:  "flow-aggregator",
+			args:          []string{"clickhouse.enable=true"},
+		},
+		{
+			name:          "valid configMap and invalid args",
+			configMapName: "testConfigMap",
+			podNamespace:  "flow-aggregator",
+			args:          []string{"clickhouse=true"},
+			expectedErr:   "unknown configuration parameter, please check antctl set flow-aggregator -h",
+		},
+		{
+			name:          "valid configMap and invalid args syntax",
+			configMapName: "testConfigMap",
+			podNamespace:  "flow-aggregator",
+			args:          []string{"clickhouse.enable=true clickhopuse.debug=true"},
+			expectedErr:   "query should contain exactly one '='",
+		},
+		{
+			name:          "wrong pod namespace",
+			configMapName: "testConfigMap",
+			podNamespace:  "flow-visibility",
+			expectedErr:   "request namespace does not match object namespace",
+		},
+	}
+
+	getClients = func(cmd *cobra.Command) (kubernetes.Interface, error) {
+		return k8sClient, nil
+	}
+
+	for _, tc := range tcs {
+		t.Run(tc.name, func(t *testing.T) {
+			buf := new(bytes.Buffer)
+			cmd.SetOutput(buf)
+			cmd.SetOut(buf)
+			cmd.SetErr(buf)
+			os.Setenv("POD_NAMESPACE", "flow-aggregator")
+			os.Setenv("FA_CONFIG_MAP_NAME", tc.configMapName)
+			err := updateRunE(cmd, tc.args)
+			if err != nil {
+				assert.Contains(t, err.Error(), tc.expectedErr)
+			} else {
+				assert.NoError(t, err)
+			}
+		})
+	}
+}
+
+func TestSetBoolOrFail(t *testing.T) {
+	tcs := []struct {
+		name        string
+		b           bool
+		value       string
+		parsedValue bool
+	}{
+		{
+			name:        "true boolean value",
+			value:       "1",
+			parsedValue: true,
+		},
+		{
+			name:        "false boolean value",
+			value:       "f",
+			parsedValue: false,
+		},
+		{
+			name:  "invalid value",
+			value: "tt",
+		},
+	}
+
+	for _, tc := range tcs {
+		t.Run(tc.name, func(t *testing.T) {
+			err := setBoolOrFail(&tc.b, tc.value)
+			if err != nil {
+				assert.Contains(t, err.Error(), "invalid syntax")
+			} else {
+				assert.Equal(t, tc.parsedValue, tc.b)
+			}
+		})
+	}
+}
+
+func TestSetStringOrFail(t *testing.T) {
+	tc := []struct {
+		b     string
+		value string
+	}{
+		{
+			value: "11mm",
+		},
+	}
+
+	_ = setStringOrFail(&tc[0].b, tc[0].value)
```

Comment:
`b` is too simple for a field name, if there is only one case, you may just remove table defined tests.